src/adv.py

```diff
@@ -38,7 +43,12 @@
 #
 
 # Make a new player object that is currently in the 'outside' room.
-
+print('Hello there!')
+new_player = input('What is your name?')
+if len(new_player) == 0:
+    print('Please enter your name.\n')
+player = Player(new_player, room['outside'])
```

Comment:
Great form here. Looks like you have a good grasp of the object square-brace notation.